Cargo.toml

```diff
@@ -8,13 +8,13 @@ categories = ["embedded", "encoding", "no-std"]
 keywords = ["serde", "cbor", "rust", "no-std"]
 repository = "https://github.com/ycrypto/heapless-bytes"
 readme = "README.md"
-edition = "2018"
+edition = "2021"
 
 [dependencies]
 typenum = "1.11.2"
 
 [dependencies.heapless]
-version = "0.7"
+version = "^0.8"
```

Comment:
Why the caret?

`version = "0.8"` does the same: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements